Makefile.docker

```diff
@@ -17,8 +17,12 @@ test: $(TEST_IMAGE)
 
 $(TEST_IMAGE):
 	$(eval IMAGE := $(subst _COLON_,:,$@))
-	$(DOCKER) pull $(IMAGE)
-	git archive HEAD | $(DOCKER) run \
-		--cap-add=SYS_ADMIN --rm -i -v /build -w /build -e PY_COLORS=1 \
-		-e TOXENV="$(TOXENV)" -e PYTEST_ADDOPTS="$(PYTEST_ADDOPTS)" $(IMAGE) \
-		/bin/sh -c 'dnf -y install tar && tar -x && dnf -y upgrade openssl-libs && dnf -y install yum && dnf -y install $$(cat test-deps-fedora.txt) && tox'
+	$(DOCKER) build \
+		--build-arg DOCKER_IMAGE="${DOCKER_IMAGE}" \
```

Comment:
> I don't know, does it? 

What is the meaning of "does it"?

The advantage of a script make code simple.

Actually the content of `Makefile.docker` is 3 statement.
If we use the bash shell script, we can write it on 5 lines.

1. if user does not belong to docker group, add "sudo".
2. docker build.
3. docker run. 

When we write .PHONY lone code target, we have to consider colon ":", as a result, the code line become longer.
So, long .PHONY target can be a scrip file.

But if you do not like it, I do not change this part.